.github/workflows/prepare-release.yml

```diff
@@ -30,9 +30,14 @@ jobs:
           git config --local user.email "x@empathy.co"
           git config --local user.name "empathy/x"
       - name: Prepare the release
-        run: "npm run prepare-release:${{ github.event.inputs.kind }}"
-      - name: Push changes
-        uses: ad-m/github-push-action@master
+        run: npm run prepare-release:${{ github.event.inputs.kind }}
```

Comment:
Does the `prepare release` with `alpha` generates de changelog anyway?
https://github.com/javieri-empathy/lerna-repo/pull/23/files

I thought this was done only on stable release

Reply:
Do those changes remains as part of the `alpha` release when doing a `stable` ?
https://github.com/javieri-empathy/lerna-repo/pull/24/files

Reply:
Yes, and yes. This is the same behavior that we currently have in x-components. I started exploring the possibility of making a script that automated updating the changelog removing the pre-release versions. I can officially start working on that if you allow me too.

Reply:
Not necessary at the moment, I only wanted to be sure with this